src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -60,19 +69,10 @@ public static void GetDirectoryName_SpaceOrControlCharsThrowOnWindows(string pat
         }
 
         [Fact]
-        public static void GetDirectoryName_SpaceThrowOnWindows_Core()
+        [ActiveIssue(27269)]
+        public static void GetDirectoryName_Space_Core()
         {
-            string path = " ";
-            Action action = () => Path.GetDirectoryName(path);
-            if (PlatformDetection.IsWindows)
-            {
-                AssertExtensions.Throws<ArgumentException>("path", null, () => Path.GetDirectoryName(path));
-            }
-            else
-            {
-                // This is a valid path on Unix
-                action();
-            }
+            Assert.Null(Path.GetDirectoryName(" "));
```

Comment:
We dont have a test for this ```"\r\n"``` on .net core

Reply:
Thanks for pointing it out. I'll be making several passes through the tests as we settle in the changes in CoreLib. Please feel free to create new tests where you see we're lacking as well. :)